packs/st2/actions/workflows/send_anonymous_install_data.yaml

```diff
@@ -0,0 +1,37 @@
+---
+  chain:
+    -
+      name: "check_for_permission_to_send_anonymous_data"
+      ref: "st2.kv.get"
+      params:
+        key: "st2::collect_anonymous_data"
+      on-success: "get_server_uuid"
```

Comment:
If anything, I'd rather have another exit status, `aborted`, with another color, because abusing success/failure like this is equally non-ideal.